### ISSUE TITLE
Add "WCAG 2 Non-Interference" to requirements mapping [80f0bf, 80af7b]

### ISSUE THREAD

**Issue by WilcoFiers** [act-rules/act-rules.github.io#1348]
Some of WCAG's SCs are part of the non-interference requirement (conformance requirement 5). The ACT Task Force discussed this yesterday and concluded that, since non-interference is an accessibility requirement, it has to be mentioned in the accessibility requirements mapping. This is a practice that Trusted tester v5 follows today as well.

I'm not sure if this is something that can be done today already, or if we need to add explicit support for this. @jkodu, can you weigh in on that?

**Comment by Jym77** [comment]
Looking at https://github.com/act-rules/act-rules-web/blob/c23d9f08cc915a66082487a53fe38638114b9265/src/components/accessibility_requirements/index.js#L94-L122 we should already be good using
>  aria11:#ua_noninterference:
    title: ARIA 1.1, 3.1 Non-interference with the Host Language
    forConformance: true
    failed: not satisfied
    passed: further testing needed
    inapplicable: further testing needed

That will display a conformance requirement similar to https://act-rules.github.io/rules/6a7281

If we need to refer to another document instead, some code need to be added to handle the key. But looking at the component above, it is already quite modular, so it should be fairly easy work (hopefully just adding a new entry to this object: https://github.com/act-rules/act-rules-web/blob/c23d9f08cc915a66082487a53fe38638114b9265/src/components/accessibility_requirements/index.js#L95 )

**Comment by Jym77** [comment]
Link to https://www.w3.org/TR/WCAG21/#cc5
